docs/rackhd2.0/configuration.rst

```diff
@@ -15,6 +15,7 @@ config.json_
         "apiServerAddress": "172.31.128.1",
         "apiServerPort": 9080,
         "broadcastaddr": "172.31.131.255",
+        "dhcpPollerActive": true,
```

Comment:
@yyscamper - the default is set to off. And if it is not in the config file it is also off.